src/App.js

```diff
@@ -19,22 +20,36 @@ function App() {
   return (
     <div className="App">
 
-      <BrowserRouter>
+      <BrowserRouter className="browser">
         <header className="header">
-          <h1>
-            SECRET FOREST
+          <h1 className="header-h1">
+            Bookstore CMS
           </h1>
           <ul className="header-ul">
             <li>
-              <Link className="header-ul-link" to="/">
-                Books
+              <NavLink
```

Comment:
- [ ] Kindly make that your design is exactly the same as in the [given design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) 😄  as it is a part of the requirements 👍. Make sure `color` , `padding`.  `margin` , etc , are exactly the same as in the given design 👍  

### Navbar
| Your Design  | Given design |
| :---: | :---: |
| ![Capture](https://github.com/khanh-devos/rjs-bookstore-khanh/assets/80458199/1feaffc6-5467-475e-ab10-cd5d034c58d8) | ![Capture 1](https://github.com/khanh-devos/rjs-bookstore-khanh/assets/80458199/ebae1bb9-5ec1-46e7-872c-f13cbcc332c3) |

### BookList
| Your Design  | Given design |
| :---: | :---: |
| ![Capture33](https://github.com/khanh-devos/rjs-bookstore-khanh/assets/80458199/d512d8ee-a0bc-4f4b-8acb-881db738cbde) | ![Capture33](https://github.com/khanh-devos/rjs-bookstore-khanh/assets/80458199/fbde0131-0276-4314-a74f-b5eefaf4a75e) |

### Add book form
| Your Design  | Given design |
| :---: | :---: |
| ![Capture4](https://github.com/khanh-devos/rjs-bookstore-khanh/assets/80458199/1f0dedbb-e47d-47af-9d06-d6d3886480e8)  | ![Capture44](https://github.com/khanh-devos/rjs-bookstore-khanh/assets/80458199/72d7ea33-c66f-42b2-aef0-13afd2734d85)
 |